src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue10987.cs

```diff
@@ -0,0 +1,40 @@
+﻿using NUnit.Framework;
+using UITest.Appium;
+using UITest.Core;
+namespace Microsoft.Maui.TestCases.Tests.Issues
+{
+	public class Issue10987 : _IssuesUITest
+	{
+		const string Editor = "Editor";
+		const string RTLEditor = "RTLEditor";
+		public Issue10987(TestDevice device) : base(device) { }
+
+		public override string Issue => "Editor HorizontalTextAlignment Does not Works.";
+
+		[Test]
+		[Category(UITestCategories.Editor)]
+		public void EditorPlaceholderRuntimeTextAlignmentChanged()
+		{
+			App.WaitForElement(RTLEditor);
+			App.Tap("Button");
+			VerifyScreenshot();
+		}
+
+		[Test]
+		[Category(UITestCategories.Editor)]
+		public void EditorRuntimeTextAlignmentChanged()
```

Comment:
Need to update the macOS snapshots. We changed the way to capture screenshots and requires new ones.

Example:
EditorRuntimeTextAlignmentChanged
`Snapshot different than baseline: EditorRuntimeTextAlignmentChanged.png (size differs - baseline is 1920x1080 pixels, actual is 789x563 pixels)`
![image](https://github.com/user-attachments/assets/4fe556b3-0c86-458d-95ca-363aa3f4f123)